RNDeviceInfo.podspec

```diff
@@ -14,6 +14,7 @@ Pod::Spec.new do |s|
 
   s.source       = { :git => "https://github.com/react-native-device-info/react-native-device-info.git", :tag => "v#{s.version}" }
   s.source_files  = "ios/**/*.{h,m}"
+  s.resource = 'ios/PrivacyInfo.xcprivacy'
```

Comment:
https://github.com/google/promises/pull/226#discussion_r1447871477 :)

Reply:
Thanks for that, updated with the change.